hw-cliargs/ls_helper.py

```diff
@@ -0,0 +1,84 @@
+"""Helper classes for ls utility"""
+
+from collections import OrderedDict
+import functools
+
+
+class EntryInfo:
+    """Class to store entries information and provide printable representation for it"""
+    def __init__(self, **kwargs):
+        self.__dict__.update(kwargs)
```

Comment:
I would use a regular property e.g. self.options = kwargs, because it is better if you want to add some usual properties in the future, but for now this approach looks nice.